masonite/response.py

```diff
@@ -87,6 +89,10 @@ def view(self, view, status=200):
 
         if isinstance(view, dict) or isinstance(view, list):
             return self.json(view)
+        else if isinstance(view, Collection):
+            return view.serialize()
+        else if isinstance(view, Model):
+            return view.to_json()
```

Comment:
You would want to return json with json headers. this line should be. 

```suggestion
            return self.json(view.to_dict())
```

Reply:
oh good catch

Reply:
The View has the `to_dict` method?

Reply:
yes it should be to_dict() since the json method needs a dictionary